media/css/115release.css

```
@@ -0,0 +1,1178 @@
+
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  max-width: var(--max-width);
+  margin: 0 auto;
+}
+
+.split {
+  flex-direction: row;
+}
+
+.split.reverse {
+  flex-direction: row-reverse;
+}
+
+.split > .info {
+  flex-basis: calc(66% - 12px);
+}
+
+.split > .image {
+  flex-basis: calc(33% - 12px);
+}
+
+.info {
+  padding: 64px 32px;
+}
+
+.slider-box {
+  background-image: url("/media/img/thunderbird/whatsnew/115/new-old-fade.png");
+  background-size: auto 95%;
+  background-position: center;
+  background-repeat: no-repeat;
+  aspect-ratio: 16 / 10;
+  width: 100%;
+  position: relative;
+  isolation: isolate;
+  opacity: 0.75;
+}
+
+.slider-box:hover,
+.slider-box:focus-within {
+  opacity: 1;
+}
+
+#before,
+#after {
+  background-size: auto 100%;
+  background-position: left center;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+}
+
+#before {
+  background-image: url("/media/img/thunderbird/whatsnew/115/102.png");
+  z-index: 0;
+}
+
+#after {
+  background-image: url("/media/img/thunderbird/whatsnew/115/115.png");
+  z-index: 1;
+  width: calc(var(--slider) - 9px);
+}
+
+.slider {
+  position: absolute;
+  -webkit-appearance: none;
+  appearance: none;
+  width: 100%;
+  height: 100%;
+  background: transparent;
+  color: transparent;
+  outline: none;
+  margin: 0;
+  z-index: 2;
+}
+
+input[type="range"]::-moz-range-thumb,
+input[type="range"]::-webkit-range-thumb {
+  -webkit-appearance: none;
+  appearance: none;
+  accent-color: transparent;
+  color: transparent;
+  width: 6px;
+  height: 105%;
+  border: none;
+  background: transparent;
+  cursor: pointer;
+  opacity: 0;
+}
+
+.slider-bar {
+  pointer-events: none;
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: transparent;
+  background-image: linear-gradient(
+    to bottom,
+    transparent,
+    var(--color-magenta-90) 20%,
+    var(--color-purple-90) 80%,
+    transparent
+  );
+  border: none;
+  width: 6px;
+  height: 105%;
+  inset-inline-start: calc(var(--slider) - 9px);
+}
+
+.slider-tab {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  pointer-events: none;
+  margin-inline: -29px;
+  margin-block-end: 32px;
+  width: 32px;
+  height: 32px;
+  border: 1px solid rgba(0, 0, 0, 0.5);
+  border-radius: 50%;
+  background-color: var(--color-ink-80);
+  background-image: linear-gradient(
+    to bottom,
+    var(--color-ink-70),
+    var(--color-ink-90)
+  );
+  z-index: 3;
+}
+
+.slider-tab::before,
+.slider-tab::after {
+  content: "";
+  padding: 4px;
+  margin-inline: 1px;
+  display: inline-block;
+  border: solid var(--color-ink-20);
+  border-radius: 1px;
+  border-width: 0 1px 1px 0;
+}
+
+.slider-tab::before {
+  transform: rotate(135deg);
+}
+
+.slider-tab::after {
+  transform: rotate(-45deg);
+}
+
+.info p {
+  color: var(--info-txt);
+  font-weight: 500;
+  font-size: 2rem;
+  margin-block: 0.5em 1em;
```

Let's tweak a bit this to improve readability and visual rhythm.
Also, since all sections have an header with an icon, we can visually indent the paragraph to match the start of the icon.
You can create an exception class in case a section doesn't have a title with an icon.
```suggestion
  font-size: 1.8rem;
  margin-block: 0.5em 1em;
  line-height: 1.35em;
  padding-inline-start: 44px;
```